raft/log_test.go

```diff
@@ -671,13 +671,13 @@ func TestIsOutOfBounds(t *testing.T) {
 			}()
 			err := l.mustCheckOutOfBounds(tt.lo, tt.hi)
 			if tt.wpanic {
-				t.Errorf("%d: panic = %v, want %v", i, false, true)
+				t.Errorf("#%d: panic = %v, want %v", i, false, true)
 			}
 			if tt.wErrCompacted && err != ErrCompacted {
-				t.Errorf("%d: err = %v, want %v", i, err, ErrCompacted)
+				t.Errorf("#%d: err = %v, want %v", i, err, ErrCompacted)
 			}
 			if !tt.wErrCompacted && err != nil {
-				t.Errorf("%d: unexpected err %v", i, err)
+				t.Errorf("#%d: unexpected err %v", i, err)
```

Comment:
may be i am missing something, but what the need for `#` here?

Reply:
@vimalk78 This is for the unification of the code style, there are `#` prefix in other test cases, like [log_test.go#136](https://github.com/etcd-io/etcd/blob/master/raft/log_test.go#L136)、[log_test.go#805](https://github.com/etcd-io/etcd/blob/master/raft/log_test.go#L805)...

Reply:
aah, i see. It is being used to prefix the test case index in the table.
Thanks, i learned something. !!